azurecompute/src/test/java/org/jclouds/azurecompute/features/VMImageApiLiveTest.java

```diff
@@ -0,0 +1,205 @@
+        cloudService = getOrCreateCloudService(CLOUD_SERVICE, LOCATION);
+
+        roleInstanceReady = retry(new Predicate<String>() {
+
+            @Override
+            public boolean apply(String input) {
+                Deployment.RoleInstance roleInstance = getFirstRoleInstanceInDeployment(input);
+                return roleInstance != null && roleInstance.instanceStatus() == READY_ROLE;
+            }
+        }, 600, 5, 5, SECONDS);
+
+        final DeploymentParams params = DeploymentParams.builder()
+                .name(DEPLOYMENT)
+                .os(OSImage.Type.LINUX)
+                .sourceImageName(BaseAzureComputeApiLiveTest.IMAGE_NAME)
+                .mediaLink(AzureComputeServiceAdapter.createMediaLink(storageService.serviceName(), DEPLOYMENT))
+                .username("test")
+                .password("supersecurePassword1!")
+                .size(RoleSize.Type.BASIC_A2)
+                .subnetName(Iterables.get(virtualNetworkSite.subnets(), 0).name())
+                .virtualNetworkName(virtualNetworkSite.name())
+                .externalEndpoint(DeploymentParams.ExternalEndpoint.inboundTcpToLocalPort(22, 22))
+                .build();
+        Deployment deployment = getOrCreateDeployment(cloudService.name(), params);
+        Deployment.RoleInstance roleInstance = getFirstRoleInstanceInDeployment(DEPLOYMENT);
+        assertTrue(roleInstanceReady.apply(DEPLOYMENT), roleInstance.toString());
+        roleName = roleInstance.roleName();
+        diskName = deployment.roleList().get(0).osVirtualHardDisk().diskName();
+    }
+
+    @Test(dependsOnMethods = "testCaptureVMImage")
+    public void testCreate() {
+        Date date = new Date();
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+        VMImage image = api().list().get(5);
+        VMImageParams.OSDiskConfigurationParams osParams = VMImageParams.OSDiskConfigurationParams
+                .OSDiskConfiguration(CREATE_IMAGE_NAME + "osdisk",
+                        VMImageParams.OSDiskConfigurationParams.Caching.READ_ONLY,
+                        VMImageParams.OSDiskConfigurationParams.OSState.SPECIALIZED,
+                        image.osDiskConfiguration().os(),
+                        URI.create(
+                                "https://" + storageService.serviceName()
+                                        + ".blob.core.windows.net/vhds/" + CAPTURED_IMAGE_NAME + "-os-" + dateFormat.format(date) + ".vhd"),
+                        30,
+                        "Standard");
+        VMImageParams params = VMImageParams.builder().name(CREATE_IMAGE_NAME).label(CREATE_IMAGE_NAME)
+                .description(image.description()).recommendedVMSize(image.recommendedVMSize())
+                .osDiskConfiguration(osParams).imageFamily(image.imageFamily())
+                .build();
+
+        String requestId = api().create(params);
+        assertNotNull(requestId);
+        operationSucceeded.apply(requestId);
```

Comment:
Is this something that needs to be deleted at some point when the test finishes?